Sources/ApolloCodegenLib/FileGenerators/OperationManifestFileGenerator.swift

```diff
@@ -57,8 +57,22 @@ struct OperationManifestFileGenerator {
   func generate(fileManager: ApolloFileManager = .default) throws {
     let rendered: String = try template.render(operations: operationManifest)
 
+    var manifestPath = config.output.operationManifest.unsafelyUnwrapped.path
+    let relativePrefix = "./"
+      
+    // if path begins with './' the path should be relative to the config.rootURL
+    if manifestPath.hasPrefix(relativePrefix) {
+      let fileURL = URL(fileURLWithPath: String(manifestPath.dropFirst(relativePrefix.count)), relativeTo: config.rootURL)
+      manifestPath = fileURL
+          .resolvingSymlinksInPath()
+          .appendingPathExtension("json")
```

Comment:
Are we forgetting to check for the existing path extension here? Glad you added it below due to Calvin's comment, but it seems like in the case of a relative path, we aren't checking for the `.json` already existing?

Unless `Swift.URL.appendingPathExtension()` already handles checking for an existing path extension for you?

Reply:
I thought the `appendingPathExtension` would check for that, but adding the tests for this can verify that for sure.

Reply:
Looks like it didn't handle that so thats fixed and tests added